### PR TITLE
DM-32869: Added a conda dependency for pytest.

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -43,3 +44,4 @@ requirements:
     - setuptools_scm
     - ts-tcpip
     - ts-utils
+    - pytest
```
Comment (tribeiro) [inline conda/meta.yaml:47]: Are you sure you want to add `pytest` to the "run" dependency? Do you really need it to run the package? Consider making the module that depends on `pytest` a submodule that is not imported by default. I am really weary of requiring "test" packages as a run dependency.

Comment (wvreeven) [reply conda/meta.yaml]: Good point, thanks! Done.

File: conda/meta.yaml [hunk 2]
```diff
@@ -24,6 +24,7 @@ test:
     - ts-conda-build =0.3
     - ts-tcpip
     - ts-utils
+    - pytest
```
Comment (tribeiro) [inline conda/meta.yaml:27]: `pytest` is part of `ts-conda-build`... you can remove it altogether from here

https://github.com/lsst-ts/ts_recipes/blob/fa6ac2c877a3a2966ea8dcaa3c39104a30500940/ts_conda_build/meta.yaml#L16

**Review by tribeiro — APPROVED** [review]
Looks good. See my inline comment but you can remove `pytest` from the conda recipe altogether. It is already part of the `ts-conda-build` meta package. Also, you may want to encapsulate the import and usage of `pytest` in `test_utils` to let users know that the module is only available for testing.

**Pull request merged**